brainscore_vision/model_helpers/activations/temporal/core/executor.py

```diff
@@ -25,9 +25,16 @@ class JoblibMapper:
     def __init__(self, num_threads: int):
         self._num_threads = num_threads
         self._pool = Parallel(n_jobs=num_threads, verbose=False, backend="loky")
+        self._not_supported = False
```

Comment:
no idea what this variable does, can we use a more descriptive name? what exactly is or isn't supported?